setup.py

```diff
@@ -51,6 +52,8 @@ def get_git_version():
                '--match', '%s*' % prefix]
 
         result = check_output(cmd).strip()
+        if sys.version_info >= (3, 0):
+            result = str(result, 'utf-8')
```

Comment:
It is unclear why you need to do this, please add a comment.

EDIT:
I get it, you need to convert `result` from binary to str to use `startswith` method